infra/templates/backend-state-setup/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "name" {
+  type        = "string"
+  description = "(Required) Specifies the human consumable label for this resource."
+  default     = ""
```

Comment:
You may want to instead omit the default completely. This will cause the TF commands to fail unless its provided as an argument explicitly. 

similar comment for other variables in the project.